controller-norbac.jsonnet

```diff
@@ -53,6 +53,19 @@ local v1beta1_Deployment(name) = kube.Deployment(name) {
                 runAsNonRoot: true,
                 runAsUser: 1001,
               },
+              volumeMounts_+: {
+                tmp: {
```

Comment:
As we're using kube.libsonnet, I'd suggest not explicitly setting the `name` field but rather:
```
volumeMounts_+: {
  tmp_volume: {
    mountPath: '/tmp',
  },
},
```
then modify below to be just:
```
volumes_+: {
  tmp_volume: kube.EmptyDirVolume(),
},
```

Reply:
@jjo thanks for the name field! I'm not a fan of hiding `empty{}` behind `kube.EmptyDirVolume()` though